hub/core/chunk_engine.py

```diff
@@ -174,8 +187,9 @@ def last_chunk_key(self) -> str:
 
     @property
     def tensor_meta(self):
+        cache = self.mem_cache or self.cache
```

Comment:
can't we just always have a `meta_cache`?

Reply:
i feel like having this statement everywhere would be kinda cumbersome

Reply:
we don't always want to have it. Only during transforms, otherwise we want to write to the actual storage.